code/modules/food/recipes_microwave.dm

```diff
@@ -925,85 +500,85 @@ I said no!
 	)
 	result = /obj/item/weapon/reagent_containers/food/snacks/chawanmushi
 
-/datum/recipe/microwave/beetsoup
+/datum/recipe/beetsoup
 	fruit = list("whitebeet" = 1, "cabbage" = 1)
 	reagents = list("water" = 10)
 	result = /obj/item/weapon/reagent_containers/food/snacks/beetsoup
 
-/datum/recipe/microwave/appletart
-	fruit = list("goldapple" = 1)
-	reagents = list("sugar" = 5, "milk" = 5, "flour" = 10)
-	items = list(
-		/obj/item/weapon/reagent_containers/food/snacks/egg
-	)
-	result = /obj/item/weapon/reagent_containers/food/snacks/appletart
-
-/datum/recipe/microwave/tossedsalad
+/datum/recipe/tossedsalad
 	fruit = list("cabbage" = 2, "tomato" = 1, "carrot" = 1, "apple" = 1)
 	result = /obj/item/weapon/reagent_containers/food/snacks/tossedsalad
 
-/datum/recipe/microwave/flowersalad
+/datum/recipe/flowersalad
 	fruit = list("harebell" = 1, "poppy" = 1)
 	items = list(
 		/obj/item/weapon/reagent_containers/food/snacks/roastedsunflower
 	)
 	result = /obj/item/weapon/reagent_containers/food/snacks/flowerchildsalad
 
-/datum/recipe/microwave/rosesalad
+/datum/recipe/rosesalad
 	fruit = list("harebell" = 1, "rose" = 1)
 	items = list(
 		/obj/item/weapon/reagent_containers/food/snacks/roastedsunflower
 	)
 	result = /obj/item/weapon/reagent_containers/food/snacks/rosesalad
 
-/datum/recipe/microwave/aesirsalad
+/datum/recipe/aesirsalad
 	fruit = list("goldapple" = 1, "ambrosiadeus" = 1)
 	result = /obj/item/weapon/reagent_containers/food/snacks/aesirsalad
 
-/datum/recipe/microwave/validsalad
+/datum/recipe/validsalad
 	fruit = list("potato" = 1, "ambrosia" = 3)
 	items = list(/obj/item/weapon/reagent_containers/food/snacks/meatball)
 	result = /obj/item/weapon/reagent_containers/food/snacks/validsalad
 	make_food(var/obj/container as obj)
```

Comment:
absolute pathing

Reply:
Absolute pathing?

Not sure what you're referring to.